README.md

```diff
@@ -1,12 +1,12 @@
-# Provision OSX
+# Provision OSX - Developer/Non-Developer
 
 Provision a new OSX machine with dev tools.
 
 ## Intro
 
-This is not a dotfiles configuration repository, nor it is a recipe repository. It is just a simple set of commands and tools to set up a developer's machine. Its aim is to reduce the initial time of installing tools and apps on a new machine.
+This is not a dotfile configuration repository, nor it is a recipe repository. It is just a simple set of commands and tools to set up a developer's/non-developer machine. Its aim is to reduce the initial time of installing tools and apps on a new machine.
```

Comment:
```suggestion
This is not a dotfiles configuration repository, nor it is a recipe repository. It is just a simple set of commands and tools to set up a developer's/non-developer machine. Its aim is to reduce the initial time of installing tools and apps on a new machine.
```

dotfiles is an adjective in this sentence. See https://dotfiles.github.io/

Reply:
I'm so embarrassed. I will never work late at night again. I am sorry for the awful change.

Reply:
@BaselAl Hey, I agree with not working late at night, but this change is an innocuous one.

In code reviews, my intention here is not to make others feel bad at all. I'm trying to be thorough in my code reviews in order to help others improve and teach them catch things themselves. Everything should be taken as a suggestion and best argument wins. Also, everyone makes mistakes and even a lot of my PR comments here and elsewhere may not be entirely correct, but if they trigger the right provocation or discussion so we improve our software or learn something ourselves, then it was worth it. Check out these articles mentioned here: https://github.com/clippings/clippings/blob/master/.github/CONTRIBUTING.md#code-review

Reply:
@hkdobrev I am very gratefull for your suggestions. I really appreciate the provided guide for reviewing code.